DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -297,25 +297,83 @@ public async Task ClearResponsesForUserAsync(int eventKey, ulong userId)
             await request.ExecuteAsync();
         }
 
-#pragma warning disable 1998 // Disable compiler warning stating that the unimplemented functions are synchronous
         public async Task<Dictionary<ulong, IEnumerable<EventResponse>>> GetSignupsByUserAsync(int eventId)
```

Comment:
I thought we concluded that it wasn't necessary to have both this and GetSignupsByResponse? 🙂

Reply:
Yep, I had forgotten about that! Removed in 6877d46 🙂